ambari-server/src/main/java/org/apache/ambari/server/controller/internal/ServiceGroupResourceProvider.java

```diff
@@ -78,6 +79,7 @@
   public static final String SERVICE_GROUP_CLUSTER_NAME_PROPERTY_ID = RESPONSE_KEY + PropertyHelper.EXTERNAL_PATH_SEP + "cluster_name";
   public static final String SERVICE_GROUP_SERVICE_GROUP_ID_PROPERTY_ID = RESPONSE_KEY + PropertyHelper.EXTERNAL_PATH_SEP + "service_group_id";
   public static final String SERVICE_GROUP_SERVICE_GROUP_NAME_PROPERTY_ID = RESPONSE_KEY + PropertyHelper.EXTERNAL_PATH_SEP + "service_group_name";
+  public static final String SERVICE_GROUP_SERVICE_GROUP_MPACKNAME_PROPERTY_ID = RESPONSE_KEY + PropertyHelper.EXTERNAL_PATH_SEP + "mpacks";
```

Comment:
Can we simply call it SERVICE_GROUP_MPACKNAME_PROPERTY_ID?

Reply:
We need to add this property to -
1. static {
     // properties
https://github.com/apache/ambari/pull/234/files#diff-b9a4d90adcd0c31956baab7c90540e53R107
2. To the response of createResourcesAuthorized
https://github.com/apache/ambari/pull/234/files#diff-b9a4d90adcd0c31956baab7c90540e53L162
3. To the response of getResourcesAuthorized
https://github.com/apache/ambari/pull/234/files#diff-b9a4d90adcd0c31956baab7c90540e53R202

Reply:
OK, I will change it to SERVICE_GROUP_MPACKNAME_PROPERTY_ID